contracts/Agreement.sol

```diff
@@ -51,7 +51,7 @@ contract Agreement is
     ////////////////////////
 
     SignedAgreement[] private _amendments;
-    mapping(address => bool) private _signatories;
+    mapping(address => uint256) private _signatories;
```

Comment:
Maybe leave a comment here that it's address -> block number mapping? or rename it to idk signatoriesToBlockNumber?

Reply:
so true